bin/plugin/lib/git.js

```diff
@@ -0,0 +1,99 @@
+const path = require( 'path' );
+const SimpleGit = require( 'simple-git/promise' );
+const os = require( 'os' );
+const { v4: uuid } = require( 'uuid' );
+
+async function clone( repositoryUrl ) {
+	const gitWorkingDirectoryPath = path.join( os.tmpdir(), uuid() );
+	const simpleGit = SimpleGit();
+	await simpleGit.clone( repositoryUrl, gitWorkingDirectoryPath, [
+		'--depth=1',
```

Comment:
Feeling confident now? 😄 

https://github.com/WordPress/gutenberg/pull/20153#discussion_r377289561

Reply:
done this for debug, thanks for catching it.